docs/_sass/_main.scss

```diff
@@ -131,7 +131,7 @@ details[open] > summary {
 
 h1,
 summary {
```

Comment:
Interesting, what exactly is a summary though? Perhaps summary is what is being used for code blocks, and thus we should actually separate out these styles?

Reply:
Okay cool, you can ignore me then :)